how-tos/releaseduty_day1.md

```diff
@@ -26,13 +26,13 @@ Therefore, you ought to be present and pay attention to conversations happening
 - **_#releng_** (public RelEng channel where many non-releaseduty topics are also discussed)
 - **_#relman_** (optional - about to be retired soon, where RelMan hangs out around usually)
 
-Moreover, you need these mailing list subscriptions:
-- [release-automation-notifications](https://groups.google.com/a/mozilla.com/forum/?hl=en#!forum/release-automation-notifications)
-- [release-automation-notifications-dev](https://groups.google.com/a/mozilla.com/forum/#!forum/release-automation-notifications-dev)
+Moreover, as releaseduty you need to *subscribe* to certain mailing lists. As a general rule of thumb, all types of sign offs should go to [release signoff mailing list](https://mail.mozilla.org/listinfo/release-signoff),
+all releng automation emails should go to [release-automation-notifications](https://groups.google.com/a/mozilla.com/forum/?hl=en#!forum/release-automation-notifications) whereas all discussion topics should go to [release drivers mailing list](https://mail.mozilla.org/listinfo/release-drivers)
+Along these bread-and-butter mailing list, you ought to follow/be added to the following ML as well:
+- [release-automation-notifications-dev mailing list](https://groups.google.com/a/mozilla.com/forum/#!forum/release-automation-notifications-dev)
```

Comment:
IMO, we don't really need to subscribe to that mailing list. It's used for staging releases, and can be accessed without any subscription

Reply:
I'll prepend an "optional" here, you're right.